x-pack/plugins/watcher/public/components/watch_status.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiIcon, EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
+import { ACTION_STATES, WATCH_STATES } from '../../common/constants';
+
+function StatusIcon({ status }: { status: string }) {
+  switch (status) {
+    case WATCH_STATES.FIRING:
+    case ACTION_STATES.FIRING:
+      return <EuiIcon type="play" color="primary" />;
+    case WATCH_STATES.OK:
+    case ACTION_STATES.OK:
+    case ACTION_STATES.ACKNOWLEDGED:
+      return <EuiIcon type="check" color="secondary" />;
+    case ACTION_STATES.THROTTLED:
+      return <EuiIcon type="clock" color="warning" />;
+    case WATCH_STATES.DISABLED:
+      return <EuiIcon type="minusInCircleFilled" color="#69707D" />; // EUI color: euiColorDarkShade
```

Comment:
Was "subdued" too hard to see? If "subdued" doesn't work, then could you submit an issue in the EUI repo asking for this to be supported via a color name like "disabled" or something?

Reply:
`subdued` works, thanks! i didn't see that as an option initially.